packages/yarnpkg-core/sources/Project.ts

```diff
@@ -2022,6 +2022,9 @@ export class Project {
   }
 
   async persistInstallStateFile() {
+    if (!v8.serialize)
```

Comment:
```suggestion
    // TODO: Remove when Nodebox supports the v8 API
    if (!v8.serialize)
```
Ideally with a link to an issue so we can track it.